lms/static/scripts/postmessage_json_rpc/server/server.js

```diff
@@ -63,6 +63,12 @@ export default class Server {
     if (!this._isJSONRPCRequest(event)) {
       return;
     }
+    // Save the last reference used to the sidebar window so we can
+    // send messages back at a later time in the lms app.
+    this._sidebarWindow = {
```

Comment:
I didn't notice this until now, but it could be confusing that this reference name has `Window` in it, but this is not a `Window`. Not a big deal, just threw me a little.

Reply:
ya, i cant think of a better name atm. hmm..